### PR TITLE
Add instructions on use of EKF2_PCOEF_* parameters

### PR DESCRIPTION
**Pull request by priseborough**
Adds instructions on use of the static pressure position error correction feature.

FYI @thomasgubler

### CODE REVIEW

File: en/advanced_config/tuning_the_ecl_ekf.md [hunk 1]
```diff
@@ -68,6 +68,26 @@ If these measurements are not present, the EKF will not start.
 When these measurements have been detected, the EKF will initialise the states and complete the tilt and yaw alignment. 
 When tilt and yaw alignment is complete, the EKF can then transition to other modes of operation  enabling use of additional sensor data:
 
+#### Correction for Static Pressure Position Error
+
+Barometric pressure altitude is subject to errors generated by aerodynamic disturbances caused by vehicle wind relative velocity and orientation. This is known in aeronautics as 'static pressure position error'. The ekf2 module that uses the ecl/EKF2 estimator library provides a method of compensating for these errors provided wind speed state estimation is active.
+
+For platforms operating in a fixed wing mode, wind speed state estimation requires either [Airspeed](###airspeed) and/or [Synthetic Sideslip](###synthetic-sideslip) fusion to be enabled.
```
Comment (hamishwillee) [inline en/advanced_config/tuning_the_ecl_ekf.md]: the anchors should look like this `(#airspeed)` (only one #)

File: en/advanced_config/tuning_the_ecl_ekf.md [hunk 2]
```diff
@@ -68,6 +68,26 @@ If these measurements are not present, the EKF will not start.
...
+
+For multi-rotors, fusion of [Drag Specific Forces](###multicopter-wind-estimation-using-drag-specific-forces) can be enabled and tuned to provide the required wind velocity state estimates.
+
+The ekf2 module models the error as a body fixed ellipsoid which specifies the fraction of dynamic pressure that is added to or subtractred from the barometric pressure before it is converted to a height etimate. See the following parameter documentation for information on how to use this feature:
```
Comment (hamishwillee) [inline en/advanced_config/tuning_the_ecl_ekf.md]: typos - subtractred  and etimate

**Comment by hamishwillee** [comment]
@priseborough Modulo the typo errors, looks pretty good.

In https://docs.px4.io/en/advanced_config/static_pressure_buildup.html would I be fine just to change the last section to point to this doc rather than the `EKF2_PCOEF_*` params?

> After modifying the hardware, you can then use the `EKF2_PCOEF_*` parameters to tune for expected barometer variation based on relative air velocity.

**Comment by thomasgubler** [comment]
@hamishwillee @priseborough looks good to me

**Comment by thomasgubler** [comment]
can we merge this?

**Comment by hamishwillee** [comment]
Yes, merged. I did a small subedit first to correct the issues I found.

**Pull request merged**